src/response.js

```diff
@@ -66,13 +67,17 @@ export default class Response {
 		return this[INTERNALS].headers;
 	}
 
+	get highWaterMark() {
+		return this[INTERNALS].highWaterMark;
+	}
+
 	/**
 	 * Clone this response
 	 *
 	 * @return  Response
 	 */
 	clone() {
-		return new Response(clone(this), {
+		return new Response(clone(this, this.highWaterMark), {
```

Comment:
If I could briefly comment on this, I've noticed that in the scenario that a cloned response gets cloned, `highWaterMark` also needs to be passed in the response options to prevent it from being `undefined`.